src/arch/helpersse2.h

```diff
@@ -107,6 +107,9 @@ static INLINE int vavailability_i(int name) {
 
 static INLINE void vprefetch_v_p(const void *ptr) { _mm_prefetch(ptr, _MM_HINT_T0); }
 
+static INLINE int vtestallones_i_vo32(vopmask g) { return _mm_movemask_epi8(g) == 0xFFFF; }
```

Comment:
Below is the reference page of _mm_movemask_epi8. It collects the most significant bit of each 8-bit element, so I need to check if the returned value is 0xffff to see if the all bits are set.

https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=movemask_&expand=3591